src/styles/card.scss

```diff
@@ -0,0 +1,110 @@
+.card {
+  width: 200px;
+  padding: 15px 14px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  font-family: "Roboto", sans-serif;
```

Comment:
You should create file like `variables.scss`, import this file in `main.scss` and place them variables for fonts, colors

Check it everywhere :)